### ISSUE TITLE
Help needed on "authentication_required" error

### ISSUE THREAD

**Issue by Stanteq** [stripe/stripe-node#884]
Hi Devs, I can't figure it out how to handle this scenario using connected accounts, 3D card authentication and future payments: 

The flow:
1. Creating Customers and Connected Accounts on users registration
2. User adding new card:
   - user calls backend to get the `client_secret` 
 `stripe.setupIntents.create({customer}); // on server`
   - user insert card details and authenticate it if needed
3. When I want to charge the customer on backend I clone the paymentMethod to be able to use on connect account
```
// Clone a payment method
const pm: any = await stripe.paymentMethods.create(
    { customer, payment_method },
    { stripeAccount }
  );

const pi = stripe.paymentIntents.create({
      amount,
      application_fee_amount: fee,
      currency: "eur",
      payment_method: pm.id,
      off_session: true,
      confirm: true,
      expand: ["charges.data.balance_transaction"],
    },
    { stripeAccount })
```
In case if card require an additional authentication on creating paymentIntents returns `authentication_required` error (which I expecting) then pass to the client: client_secret and payment_method id for re-authentication.

4.  Now during authentication process on client side I got the error:

```
The provided PaymentMethod was previously used with a PaymentIntent without Customer attachment, shared with a connected account without Customer attachment, or was detached from a Customer. It may not be used again. To use a PaymentMethod multiple times, you must attach it to a Customer first.
```
So I suppose that I need to create first a customer (under connected account) and attach that payment method. But this means that on every charge I need to create a new customer. What is the correct way to achieve this?

**Comment by remi-stripe** [comment]
@Stanteq Github issues should be limited to bugs or feature requests with the library itself. Here, you have an integration question though so we recommend that you talk to our support team instead for this. You can contact them here: https://support.stripe.com/contact

To unblock you though, the issue here is that this payment requires 3DS. What you simply need to do is confirm that specific PaymentIntent client-side instead. Right now it's blocked because the bank required 3DS for this payment. The way you do this is to:
1/ Clone the PaymentMethod again server-side to get a new version of it.
2/ Use the `client_secret` for that specific `PaymentIntent` client-side and pass that new PaymentMethod id (pm_123) when confirming on behalf of the connected account.

If this succeeds, you're all set. If it doesn't then you would collect new card details to confirm that PaymentIntent instead.

**Comment by Stanteq** [comment]
Thx, sounds very promising. Sorry for the offtopic but https://support.stripe.com/contact doesn't helped me too much.

**Comment by remi-stripe** [comment]
All good, we also help developers on IRC on Freenode in the #stripe channel. There, you can talk to other developers using Stripe and some Stripes support engineers.
